File: InterviewQuestions/Javascript Interview Programming Questions/task10.js
// Write a Javascript program to check a number is Armstrong or not


let number = 1634

let new_number = String(number)

let n = 0

let sum = 0

while(n < new_number.length) {

    value = Math.pow(Number(new_number[n]), new_number.length) 
    
    sum = sum + value

    n = n + 1
}

if (sum === number) {
    console.log("Armstrong Number")
}
else {
    console.log("Not an Armstrong Number")
}

File: InterviewQuestions/Python Interview Programming Questions/task10.py
# Write a Python program to check a number is Armstrong or not

number = 153

string_number = str(number)

total = 0

for a in string_number:

    value = str(string_number) ** len(a)

    total = total + value

if number == total:

    print(f"The number ${number} is Armstrong number")

else:

    print(f"The number  ${number} is not Armstrong number")